src/pages/Chat.vue

```diff
@@ -24,7 +24,7 @@
 
     <q-page-container>
       <q-page>
-        <q-scroll-area ref="chatScroll" class="scroll-area-bordered q-px-none absolute full-width full-height">
+        <q-scroll-area ref="chatScroll" :class="`scroll-area-bordered q-px-none absolute full-width full-height`">
```

Comment:
what does the quote and tick do here?

Reply:
It executes it as javascript, I'm not sure how this made its way in here though.